### ISSUE TITLE
No connection

### ISSUE THREAD

**Issue by Michas0206** [retrocomputacion/retroterm#9]
Hi again,
now I am able to open Rapidfire-BBS through Retroterm, so parts seem to work.
Unfortunately I do not get a connection to a running RetroBBS-server running on a Raspberry next room in the same network connected by wifi to the same router as the C64 is cabled to.

![IMG_20241021_002715_DRO](https://github.com/user-attachments/assets/2172d661-54f8-4d72-a44f-99c6262c13b8)
![IMG_20241021_002729_DRO](https://github.com/user-attachments/assets/740cae36-3930-499d-bc8e-90eeaf61a22b)
![IMG_20241021_002747_DRO](https://github.com/user-attachments/assets/0e8569bc-f549-4155-ba85-2acc1954ab69)

In the last few hours I tried around with "exposed host" settings in many variations, without success, too.
![Screenshot_20241021-002822_Firefox Beta](https://github.com/user-attachments/assets/0c48926b-a254-4820-b8ef-7102f2414621)

What am I doing wrong? What exact adress have I to call from C64 side? Any special settings I have to do in the router? Port is set to 6488 here today just testwise becaus it did not work with 6400 yesterday.
Greets, Michael

**Comment by retrocomputacion** [comment]
The IP should be the one router assigns to the RPi, so the one youre using should work.
You'll have to setup a virtual server/port forwarding on your router, using the RPi IP address and the port(s) you want to expose, in your case that would be 6488.
I did a little search and I think this functionality is called 'Port sharing' in your router brand.

Let me know how it goes

**Comment by Michas0206** [comment]
Whatever I do, I dont get it to work unfortunately.
I have to investigate traffic more - but dont know where to start exaclty.

All I get is "no carrier" and "no answer", most times immediately, simetimes it takes a few seconds.

I need a way to find out if " something" reaches the Raspberry at all, or where the request "atd" from the C64 gets stuck exactly - but how...seems I need a router-expert to analyze the traffic in the FritzBox...but I dont know hot to get such details or what I could try now.

![Screenshot_20241026-221803_Firefox Beta](https://github.com/user-attachments/assets/65a3499b-b1c8-461a-90a0-159b43d7d740)
![IMG_20241026_221947_DRO](https://github.com/user-attachments/assets/c0396100-30ab-4958-9f57-b0a7b0f7b94b)
![IMG_20241026_221821_DRO](https://github.com/user-attachments/assets/0c3908d4-1659-4624-8a80-549642c53f13)

![IMG_20241026_221845_DRO](https://github.com/user-attachments/assets/9d5c1e28-802f-4833-b57b-f906545082c3)


Protocol for port sharing I have chosen TCP, but also tried UDP, exposed host, exposed host for C64 and Ports from 6400-6500 for Raspberry...always the same result.

 On raspberry side a far as I know there is no firewall.

**Comment by Michas0206** [comment]
Rapidfire-BBS works immediately without any problem:
![Uploading IMG_20241026_223655_DRO.jpg…]()

I hope you have an idea how I could check and analyze whats going on here a bit more / maybe on router/Raspberry-side....I am a complete network/telnet/bbs-noob, isnt there any "ping" command or some way to check where the traffic gets stuck?

**Comment by retrocomputacion** [comment]
Only thing I find could be wrong is that you seem to be opening the 6400-6500 port range only for IPv6 connections

**Comment by Michas0206** [comment]
You're right, but even If I correct this, always the same result, whatever I do.

Rapidfire bbs works, but I recognized that all 'Turbo56k"-bbs adresses I can find do not work on Retroterm here - are they all currently down - or should they work?
Do you have an adress of a Turbo56k bbs, that currently definately is up and working?
Just to figure out where the problem is (and I already have had firmware-problems on Ultimate-side).

In the meanwhile I installed Wireshark on the Raspberry and see, that on wifi definately a date package goes in from Ultimate-IP when I do a "atd...". Not sure what happens then, there is a deep red answer, thats it. But as traffic goes in over wifi from correct ip, I assume router and port-things are fine.

**Comment by retrocomputacion** [comment]
This is a live status monitor for retroBBS instances: https://solidstate.com.ar/bbsstatus/

About your problem, try changing the BBS ip in the config.ini back to 0.0.0.0 or 127.0.0.1 (Usually the default 0.0.0.0 works, but some OSs need to use another localhost alias)

**Comment by Michas0206** [comment]
Many thanks, these bbs work at least. I'll do more tests tomorrow with my network.
